src/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Drawer } from '../index.js'
+import { Story } from '@storybook/react'
+import styled from 'styled-components'
+
+export default {
+  title: 'Drawer',
+  component: Drawer,
+}
```

Comment:
this also could be typed -> `as Meta<React.ComponentProps<typeof AppBar>>`

Reply:
did not know that will update